core/src/mindustry/world/blocks/production/Separator.java

```diff
@@ -30,6 +30,7 @@ public Separator(String name){
         solid = true;
         hasItems = true;
         hasLiquids = true;
+        sync = true;
```

Comment:
doesn't sync the seed when initially placed

Reply:
How about making the seed depend on the tile position, then? Initialize it in `created`, with something like `seed = Mathf.randomSeed(tile.pos(), 0, Integer.MAX_VALUE - 1)`

Reply:
@Anuken your code sample looks fine & implemented as-is